### DISCUSSION TITLE
#42 - Марк Стручков - курс на Яндекс Практикуме, собеседования и библиотека Кваркус

### DISCUSSION THREAD

**Discussion by giscus** [javaswag/javaswag.github.io#3] [Announcements]
# #42 - Марк Стручков - курс на Яндекс Практикуме, собеседования и библиотека Кваркус

В 42 выпуске подкаста Javaswag поговорили с Марком Стручковым о курсах на Яндекс Практикуме, собеседованиях и Кваркусе

http://localhost:1313/episode/42/

**Comment by benchdoos** [comment]
К теме о публикации в нексус - они поменяли подход к публикации библиотек под пакетом `com.github`, всех перевезли на `io.github`. И нужно снова подтверждать свой репозиторий в GitHub, создавая временный репозиторий с номером issue в Jira.
